tests/framework/extensions/etcdsnapshot/config.go

```diff
@@ -0,0 +1,10 @@
+package etcdsnapshot
+
+const (
+	ConfigurationFileKey = "snapshotInput"
+)
+
+type Config struct {
+	UpgradeKubernetesVersion string `json:"upgradeKubernetesVersion" yaml:"upgradeKubernetesVersion"`
+	SnapshotRestore          string `json:"snapshotRestore" yaml:"snapshotRestore"`
```

Comment:
I just thought of this, so it could be a 'future works' to this suite but I think we should add 'upgradeStragegy' here as well. Either as a boolean and use preset values, or better yet as the correct upgradeStrategy values that would be used. That way a user could run this dynamically, or in the static use case it could be generated for the user like we do with the other values. 

I think this is the only value that etcd would need that the user can't manually/dynamically specify at this time.

Reply:
Fair point, that can be something we do in a separate PR as part of broader efforts to this test suite.

Reply:
Added this to the etcd snapshot epic issue so we can track it.